Sources/StreamChatUI/Utils/VideoPreviewLoader.swift

```diff
@@ -6,21 +6,32 @@ import AVKit
 import StreamChat
 import UIKit
 
-/// A protocol the video preview uploader implementation must conform to.
-public protocol VideoPreviewLoader: AnyObject {
+/// A protocol the video loader must conform to.
+public protocol VideoLoading: AnyObject {
     /// Loads a preview for the video at given URL.
     /// - Parameters:
     ///   - url: A video URL.
     ///   - completion: A completion that is called when a preview is loaded. Must be invoked on main queue.
     func loadPreviewForVideo(at url: URL, completion: @escaping (Result<UIImage, Error>) -> Void)
+    
+    /// Returns a video asset with the given URL.
+    ///
+    /// - Returns: The video asset.
+    func videoAsset(at url: URL) -> AVURLAsset
+}
+
+public extension VideoLoading {
+    func videoAsset(at url: URL) -> AVURLAsset {
+        .init(url: url)
+    }
 }
 
-/// The `VideoPreviewLoader` implemenation used by default.
-final class DefaultVideoPreviewLoader: VideoPreviewLoader {
+/// The `VideoLoading` implemenation used by default.
```

Comment:
Sorry for the nitpick but 'implementation' is the correct spelling here 👍

Reply:
✅ Fixed